build-contracts/basics/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM yolean/node as build
+
+RUN apt-get update && \
+  apt-get install make g++ python libsasl2-2 libsasl2-dev libssl-dev zlib1g-dev --no-install-recommends -y
+
+WORKDIR /kafka-cache
+
+COPY package.json .
+
+RUN npm install
+
+COPY . .
+
+
+FROM yolean/node
+
+# Without this I'm unable to get things running due to missing
+# c libs like libssl.so.1.1
+RUN apt-get update && \
+  apt-get install libsasl2-2 libsasl2-dev libssl-dev zlib1g-dev --no-install-recommends -y
```

Comment:
It's perfectly ok to add runtime dependencies. If they're build time dependencies we typically try to do the build in the same step and clean up afterwards. Here you might simply clean up after apt-get update using something like https://github.com/Yolean/docker-base/blob/master/node/Dockerfile#L14.

Reply:
We'll probably want node-rdkafka in a lot of services, so maybe we should create a `yolean/node-rdkafka` common base image? The versions of these packages won't change unless we upgrade debian.

Reply:
They're probably buildtime deps, but as this isn't ever gonna get pushed (it's a test) I thought I wouldn't put that much effort into cleaning up.

yolean/node-rdkafka is probably a good idea. Not 100% sure it would look though. `usr/src/app/node_modules/node-rdkafka` with an installed and built node-rdkafka lib maybe?

Reply:
Sorry, I didn't see that it was a test build.

> yolean/node-rdkafka is probably a good idea. Not 100% sure it would look though. usr/src/app/node_modules/node-rdkafka with an installed and built node-rdkafka lib maybe?

Sound like what I had in mind. We'll get used to a specific version of the node lib and the compiled librdkafka, and make upgrades a big deal. There's lots of nitty gritty here that impacts robustness, similar to our situation with socket.io.